### DISCUSSION TITLE
Tuts or implementation from https://mx.puma.com/

### DISCUSSION THREAD

**Discussion by maleksandr** [scandipwa/scandipwa#2208] [Ideas]
I guess Scandi should provide Price Range Slider and tuts or variation for Mobile navigation in a header as on https://mx.puma.com/
Because it's strange that Scandi doesn't have  Price Range Slider in its default theme.

**Comment by yeegor** [comment]
Duplicate of https://github.com/scandipwa/scandipwa/discussions/2140
